packages/manager/modules/support/src/new-ticket/new-ticket.service.js

```diff
@@ -0,0 +1,34 @@
+export default class SupportNewTicketService {
+  /* @ngInject */
+  constructor($translate, OvhApiMe, OvhApiSupport) {
+    this.$translate = $translate;
+    this.OvhApiMe = OvhApiMe;
+    this.OvhApiSupport = OvhApiSupport;
+  }
+
+  static get name() {
+    return 'SupportNewTicketService';
```

Comment:
This is weird that the service knows it's own name, doesn't it ?

My way to tackle this is to 

````js
export const name = 'SupportNewTicketService';

export const definition = class SupportNewTicketService {}

export default {
  definition,
  name,
}
  
````